blend-library/src/test/java/com/wealthfront/TestExtensions.kt

```diff
@@ -0,0 +1,40 @@
+@file:Suppress("UNCHECKED_CAST")
+
+package com.wealthfront
+
+import org.mockito.ArgumentCaptor
+import org.mockito.Mockito
+import org.mockito.Mockito.`when`
+import org.mockito.stubbing.OngoingStubbing
+import org.mockito.stubbing.Stubber
+
+fun <T> whenever(x: T): OngoingStubbing<T> = `when`(x)
+
+fun <T> Stubber.whenever(x: T): T = `when`(x)
+
+// Black magic: do not touch unless you're a wizard
```

Comment:
Should we remove this comment and only leave the explanation on the next line? :)

Reply:
Also, can we make this internal so that no can access these outside the blend module?

Reply:
Done, and technically these are in the test folder so they shouldn't propagate outside of the module, but done anyway